OpenUpSA/wazimap-ng-ui
# Sort data mapper menu data according to order field

## Description

Sort datamapper menu data before rendering

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-277


## How is it tested automatically?


## How should a reviewer test it locally
Go to Capetown on profile youth explorer
Open datamapper
Check if order of categories, subcategories and profile indicator as according to the order defined in admin panel

PS: Issue only arises when there is data for multiple versions

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
